locksmith/migrations/20220406171811-create-refresh-tokens.js

```diff
@@ -28,6 +28,6 @@ module.exports = {
     await queryInterface.addIndex('RefreshTokens', { fields: ['nonce'] })
   },
   async down(queryInterface, Sequelize) {
-    await queryInterface.dropTable('Sessions')
+    await queryInterface.dropTable('RefreshTokens')
   },
```

Comment:
Older artifact?

Reply:
We never had Sessions table. It was likely missed in renaming Sessions to the RefreshTokens table for clarity. Shouldn't impact the migration because it is only down migration.